ChartIQ/finsemble-notifications
# Removing defunct files and update dot-net examples to works with 5.4.0

Strips down project to just the examples and corrects them (mostly .Net) to work with 5.4.0. Readme still needs updating.

Testing steps:
- Check out a fresh copy of the seed project
- Follow install instructions
- Launch Finsemble and test Notify, Subscriber and WPFNotifyComponent all work
